### PR TITLE
Now using spawn

### PR DESCRIPTION
**Pull request by plehegar**
Fixes https://github.com/w3c/spec-generator/issues/22

### CODE REVIEW

**Comment by plehegar** [comment]
I also increased the timeOut by 2 secs since webaudio is on the slow side...

File: generators/respec.js [hunk 1]
```diff
@@ -22,17 +22,34 @@ exports.generate = function (url, params, cb) {
             timedout = true;
             cb({ status: 500, message: "Processing timed out." });
         }
-    ,   10000
+    ,   12000
```
Comment (deniak) [inline generators/respec.js]: For large spec like webaudio, I'm getting regular `time out` even with `15000`, but I'm not in the US. How about making it `20s`?
Apart from that, the change LGTM.

Comment (plehegar) [inline generators/respec.js]: as long as echidna is patient enough, we should be fine.

**Comment by tripu** [comment]
OK to remove this branch?

**Pull request merged**